### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by dareno** [Starefossen/node-slackr#1]
I'm getting the following error when using slackr: `UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 39): Error: connect ETIMEDOUT`

I don't know much about javascript, but it seems that when using a `Promise` you should catch errors or you may get this exception.

I believe this is the offending line of code: https://github.com/Starefossen/node-slackr/blob/master/index.js#L6

**Comment by Starefossen** [comment]
You are correct, however, since this module returns a `Promise` it is up to the user (you) to catch any errors like this:

```js
slackr(data).then(response => {
  console.log(response.code, response.body);
}).catch(error => {
  console.error(error);
});
```

This should probably be added to the README.
